README.md

```diff
@@ -1,102 +1,95 @@
-# lint-prepush    
+# lint-prepush
+
 [![npm version](https://badge.fury.io/js/lint-prepush.svg)](https://www.npmjs.com/package/lint-prepush)
 [![npm downloads](https://img.shields.io/npm/dt/lint-prepush.svg)](https://www.npmtrends.com/lint-prepush)
 [![GitHub license](https://img.shields.io/github/license/theenadayalank/lint-prepush.svg)](https://github.com/theenadayalank/lint-prepush/blob/master/LICENSE)
 [![Build Status](https://travis-ci.org/theenadayalank/lint-prepush.svg?branch=master)](https://travis-ci.org/theenadayalank/lint-prepush)
 
+> Run linters on committed files of a GIT Branch
 
-> Run linters on committed files of a GIT Branch🔬
-
-## Getting Started 🔮
+## Getting Started
 
 This package will run linters on your project for the committed files in your branch.
 
-### Prerequisites🔭
-
-It require Node.js v6 or newer. It also requires a package to manage git hooks. I strongly suggest [Husky](https://github.com/typicode/husky) which I use for most of my projects.
-
-
-```js
-// package.json
-{
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm test",
-      "pre-push": "npm test",
-      "...": "..."
-    }
-  }
-}
-```
-
-### Installing
+### Prerequisites
 
-* This project requires a package to manage git hooks depending. 
-* I strongly suggest [Husky](https://github.com/typicode/husky) which pauses the git push by overriding pre-commit hook (it almost overrides all hooks, here we need only pre-push hook) and allow us to run our custom scripts and resumes pushing.
+- Node.js `>=6.0.0`
+- a separate package to manage integration with git hooks, like [husky](https://github.com/typicode/husky)
 
+### Installation
 
-### npm
+Install `lint-prepush` and `husky` via [`npm`](https://www.npmjs.com/):
 
-```bash
+```shell
 npm install --save-dev husky lint-prepush
 ```
 
-or using [`yarn`](https://yarnpkg.com/):
+You can also use [`yarn`](https://yarnpkg.com/) to install them:
 
-```bash
+```shell
 yarn add --dev husky lint-prepush
 ```
 
 ### Usage
 
-Configure the following scripts in package.json to lint your committed files 🔧. You can also follow any of the  [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) methods to configure lint-prepush.
+Add the following to your `package.json` to lint your committed files. You can also
+follow any of the [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) methods
+to configure `lint-prepush`.
 
-```diff
```

Comment:
The `diff` styling looked nice but made it harder to copy+paste the configuration into a user's `package.json`. Having added `lint-prepush` to several projects lately, I felt this pain when first integrating it.

I think having this as regular JSON that a user can quickly copy+paste into their `package.json` is a better way to display the information.